packages/central-server/src/apiV2/export/index.js

```diff
@@ -17,6 +18,7 @@ exportRoutes.get('/download/:fileName', useRouteHandler(DownloadHandler));
 
 exportRoutes.use(emailAfterTimeout(constructExportEmail));
 
+exportRoutes.get('/optionSets/:optionSetId', catchAsyncErrors(exportOptionSet));
```

Comment:
Batch exporting not in scope of this ticket, so no `/optionSets` endpoint without specifying an ID